flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/flight_model.cfg

```diff
@@ -218,7 +218,6 @@ aileron_to_rudder_scale = 0 ; non-dimensional
 flap_to_aileron_scale = 0 ; non-dimensional
 fly_by_wire = 0 ; Fly-by-wire available true/false
 elevator_elasticity_table = 0:1, 90:1, 200:0.5, 400:0.33
-aileron_elasticity_table = 0:1, 90:1, 200:0.5, 400:0.33
 rudder_elasticity_table = 0:1, 85:1, 110:0.7, 445:0.136
```

Comment:
The roll law changes that were added also contained adjustments to the turn coordination scheduled gain, which seemed to be sufficiently compensate for the removal of the rudder elasticity tables. In this PR, only the aileron elasticity table is removed, which would obviously lead to issues with the turn coordination. So, the rudder elasticity table should probably be removed here as well (or replaced, see comment above).

Reply:
The rudder elasticity table was originally set up to model the rudder travel limiter for the basic (non-fbw) airplane flight model. I don't know how well it worked even prior to this pr with the fbw roll law, but it pretty clearly doesn't help or isn't needed with this pr. I flew some test cases, and although the max rudder deflection does not follow the limiter curve with or without the rudder elasticity table, my few observations seemed to indicate the max rudder deflection capability is a bit better without it. (One follow-on question may be whether the rudder limiter curve should be modeled better.)

In both cases, I still recommend replacing the tables such that 1 is explicitly stated rather than removing them and depending on the MSFS default value.

Reply:
It is intended as it is. The turn coordination was not working properly anymore. So it was updated to work again with the configuration as it is now. Removing the rudder elasticity table without further adjustments would not work.

Reply:
Ok, if the turn coordination still works with both the elasticity tables and scheduled gains then it's obviously fine.